src/pages/costModelsDetails/components/dialog.tsx

```diff
@@ -45,22 +49,29 @@ const DialogBase: React.SFC<Props> = ({
       {actionText}
     </Button>
   );
-  const CancelButtonPrimary = (
+  const CloseButtonPrimary = (
     <Button
-      key="cancel"
+      key="close"
       variant="primary"
       onClick={onClose}
       isDisabled={isProcessing}
     >
-      {t('dialog.cancel')}
+      {t('dialog.close')}
     </Button>
   );
   const actions =
     actionText !== ''
       ? [CancelButtonSecondary, ProceedButton]
-      : [CancelButtonPrimary];
+      : [CloseButtonPrimary];
   return (
-    <Modal title={title} isOpen={isOpen} onClose={onClose} actions={actions}>
+    <Modal
+      isLarge={isLarge}
+      isSmall={isSmall}
```

Comment:
I see `isSmall` used in costModelsTable.tsx. Is `isLarge` still being used anywhere?

Reply:
No I will remove it.